dex/pair/src/fee.rs

```diff
@@ -38,14 +39,14 @@ pub trait FeeModule:
 
     #[endpoint(whitelist)]
     fn whitelist_endpoint(&self, address: ManagedAddress) {
-        self.require_permissions();
+        self.require_caller_has_owner_or_admin_permissions();
```

Comment:
I'm not sure if we want the user to whitelist any address they want. Maybe owner-only here.

Reply:
Done